kcidb/__init__.py

```diff
@@ -252,11 +252,31 @@ def query_main():
 def schema_main():
     """Execute the kcidb-schema command-line tool"""
     sys.excepthook = misc.log_and_print_excepthook
-    description = 'kcidb-schema - Output latest I/O JSON schema'
+    description = 'kcidb-schema - Output latest or older I/O JSON schema'
     parser = misc.OutputArgumentParser(description=description)
+    schema_major_versions = []
+    version = io.schema.LATEST
+    while version:
+        schema_major_versions.append(version.major)
+        version = version.previous
+    parser.add_argument(
+        'schema_version',
+        metavar='Schema Version',
+        type=int,
+        help='KCIDB schema version. Default is 4',
+        nargs='?',
+        default=4,
```

Comment:
Instead of hard-coding the latest version number in here, which will require updating later (and could be forgotten), could you just leave the default unspecified, and use the latest version, if the argument is not set?

Or, maybe as a better alternative, could you take the actual latest version number from `json.schema.LATEST` and put it here and in the help message?

Reply:
@spbnick kindly review